prometheus-client/src/Prometheus.hs

```diff
@@ -34,7 +34,7 @@ module Prometheus (
 -- A Counter is typically used to count requests served, tasks completed,
 -- errors occurred, etc.
 --
--- >>> myCounter <- counter (Info "my_counter" "An example counter")
+-- >>> myCounter <- register $ counter (Info "my_counter" "An example counter")
```

Comment:
I haven't got through all of the revisions yet, but there should probably be a changelog entry for this. Bonus points for one that gives concrete instructions on how to upgrade.

Reply:
Oh for sure! I usually write those after-the-fact, once the changes are finalized. 👍 on migration tips.